deploy/routeros.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+
+#Here is a script to deploy cert to routeros router.
+
+#returns 0 means success, otherwise error.
+
+########  Public functions #####################
+
+#domain keyfile certfile cafile fullchain
+routeros_deploy() {
+  _cdomain="$1"
+  _ckey="$2"
+  _ccert="$3"
+  _cca="$4"
+  _cfullchain="$5"
+
+  _debug _cdomain "$_cdomain"
+  _debug _ckey "$_ckey"
+  _debug _ccert "$_ccert"
+  _debug _cca "$_cca"
+  _debug _cfullchain "$_cfullchain"
+
+  if [ -z "$ROUTER_OS_HOST" ]; then
+    _debug "Using _cdomain as ROUTER_OS_HOST, please set if not correct."
+    ROUTER_OS_HOST="$_cdomain"
+  fi
+
+  if [ -z "$ROUTER_OS_USERNAME" ]; then
+    _err "Need to set the env variable ROUTER_OS_USERNAME"
+    return 1
+  fi
+
+  _info "Trying to push key '$_ckey' to router"
+  scp "$_ckey" "$ROUTER_OS_USERNAME@$ROUTER_OS_HOST:$_cdomain.key"
+  _info "Trying to push cert '$_ccert' to router"
+  scp "$_ccert" "$ROUTER_OS_USERNAME@$ROUTER_OS_HOST:$_cdomain.cer"
+  _info "Trying to push ca cert '$_cca' to router"
+  scp "$_cca" "$ROUTER_OS_USERNAME@$ROUTER_OS_HOST:$_cdomain.ca"
+  # shellcheck disable=SC2029
+  ssh "$ROUTER_OS_USERNAME@$ROUTER_OS_HOST" bash -c "'
+
+/certificate remove $_cdomain.cer_0
```

Comment:
I will try to incorporate this in the README, do you have any idea on how the indexing works on RouterOS? Will the latest added certificate always have 0, or is it always at the end?

Reply:
If the router doesn't already have any certs, the new one becomes item zero. Otherwise, it becomes the "next number in line". And if you remove an item, they don't get re-numbered until the next `print` command (so if the table has five items and you delete #2, the remaining items are "0,1,3,4" until you `print` them, and then they become "0,1,2,3". It makes sense when you're working interactively, but for scripting, not so much.

This is why I wish that the `/certificate import` command allowed you to set the name during import, so that the numbers didn't matter. :)